daffodil-tdml/src/test/resources/tdml-with-embedded-schema-errors.tdml

```diff
@@ -32,8 +32,7 @@
       <tdml:documentPart type="text"><![CDATA[abcdef]]></tdml:documentPart>
     </tdml:document>
     <tdml:errors>
-      <tdml:error>xs:format</tdml:error>
-      <tdml:error>invalid dfdl annotation</tdml:error>
+      <tdml:error>'import' tag should be added to</tdml:error>
```

Comment:
This is a negative test case where the errors being reported just are different now. It complains now about fn:string, and can't get to where it complains about xs:format.